_studio/shared/umc/codec/h264_dec/src/umc_h264_dec_defs_yuv.cpp

```diff
@@ -80,27 +79,32 @@ void H264DecYUVBufferPadded::Init(const VideoDataInfo *info)
 
     m_color_format = info->GetColorFormat();
     m_chroma_format = GetH264ColorFormat(info->GetColorFormat());
-    m_lumaSize = info->GetPlaneInfo(0)->m_ippSize;
+    m_lumaSize = info->GetPlaneInfo(0) ? info->GetPlaneInfo(0)->m_ippSize : mfxSize();
```

Comment:
Does it have any sense without planes? Surely no. It shouldn't come here in this case.

Reply:
May be better init member m_lumaSize by default value (0) in Init() method, if no planes?
Otherwise what we should do in case of GetPlaneInfor(0) == nullptr?
we cant return error,
keep m_lumaSize not initialized, I guess not very good idea,
may by throw an exception - I'm not sure

Reply:
Zero-sized plane is very different with absent plane. 
Just remove check as it was before. Invalid VideoDataInfo can't come here. If some future modification allow it then unhandled exception would be proper reaction.